.github/workflows/reusable-uki-test.yaml

```diff
@@ -107,11 +118,12 @@ jobs:
             --output-dir /result --keys /keys --output-type container \
             dir:/unpacked
 
-          docker load -i build/*.tar
           image=$(docker load -i build/*.tar | grep "Loaded image" | awk -F: '{ st = index($0,":");print substr($0,st+1)}' | xargs)
           rand=$(head /dev/urandom | tr -dc a-z0-9 | head -c 7)
-          temp_image=ttl.sh/$(echo $image | sed "s/:latest/-${rand}:6h/")
-          docker tag $image $temp_image
+          temp_image=quay.io/kairos/ci-temp-images:$(echo $image | sed "s/:latest/-${rand}/")
```

Comment:
This might break if tags are more restrictive that image names in regards to allowed characters (I don't know, I'm just saying). We'll see.